server/routes/index.js

```diff
@@ -0,0 +1,65 @@
+var userRoute = require('./user_route'),
+  User = require('../models/user'),
+  documentRoute = require('./document_route'),
+  roleRoute = require('./role_route'),
+  jwt = require('jsonwebtoken'),
+  config = require('../../config/config');
+
+module.exports = function (router) {
+  router.post('/users/login', function (req, res) {
```

Comment:
The user login method should be in the users controller. Avoid having logic in your routes files.